### PR TITLE
Fix doc. Symfony Finder exclude() can't handle files (only directories).

### PR DESCRIPTION
**Pull request by SebSept**
exclude handles only directories, not files.

https://github.com/symfony/symfony/blob/f6dc8260eb91c5fa2ea921384f4a9101308cea5d/src/Symfony/Component/Finder/Finder.php#L691

https://symfony.com/doc/3.4/components/finder.html#location

Am I wrong ?

### CODE REVIEW

File: src/Command/UpdateLicensesCommand.php [hunk 1]
```diff
@@ -130,8 +130,8 @@ protected function configure()
                 'exclude',
                 null,
                 InputOption::VALUE_REQUIRED,
-                'Comma-separated list of folders and files to exclude from the update',
-                implode(',', static::DEFAULT_FILTERS)
+                'Comma-separated list of folders to exclude from the update',
+                implode(',', self::DEFAULT_FILTERS)
```
Comment (Quetzacoalt91) [inline src/Command/UpdateLicensesCommand.php:134]: Looks like this line is a partial revert of https://github.com/PrestaShopCorp/header-stamp/pull/27, and having the `static::` was a good thing.

Is there any reason for this change?

**Review by Quetzacoalt91 — APPROVED** [review]
Thanks for pointing this out, following the documentation it seems you're right.

I've one comment before merging.

File: src/Command/UpdateLicensesCommand.php (hunk 1 above)
Comment (SebSept) [reply src/Command/UpdateLicensesCommand.php:134]: I can't remember. This could be skipped.

Comment (SebSept) [reply src/Command/UpdateLicensesCommand.php:134]: oops, merged bad branch.
I close this and reopen with a clean PR.

**Pull request closed without merging**